src/components/ProfileDropdown.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import {
+  Avatar,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import NewIcon from '@material-ui/icons/NewReleases';
+import BookmarkIcon from '@material-ui/icons/Bookmark';
+import DoneIcon from '@material-ui/icons/Done';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+
+// hacky way to override Menu style
+const StyledMenu = withStyles((theme) => ({
+  paper: {
+    border: '1px solid black',
+    width: '350px',
+    //background: theme.palette.background.default,
+  },
+  // placing dropdown menu below the avatar
+}))((props) => (
+  <Menu
+    elevation={0}
+    getContentAnchorEl={null}
+    anchorOrigin={{
+      vertical: 'bottom',
+      horizontal: 'center',
+    }}
+    transformOrigin={{
+      vertical: 'top',
+      horizontal: 'center',
+    }}
+    {...props}
+  />
+));
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(2),
+  },
+  menuList: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(1),
+  },
+  profilePic: {
+    alignSelf: 'center',
+  },
+  medium: {
+    width: '48px',
+    height: '48px',
+  },
+  large: {
+    width: '60px',
+    height: '60px',
+  },
+}));
+
+export default function ProfileDropdown(props) {
+  const classes = useStyles();
+  const { signOut, name, email, photo } = props;
+  const [dropMenu, setDropMenu] = React.useState(null);
+
+  const handleClick = (event) => {
+    setDropMenu(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setDropMenu(null);
+  };
+
+  return (
+    <Container>
+      <Avatar src={photo} onClick={handleClick} className={classes.large} />
+      <StyledMenu
+        anchorEl={dropMenu}
+        keepMounted
+        open={Boolean(dropMenu)}
+        onClose={handleClose}>
+        <Grid container spacing={2} className={classes.root}>
+          <Grid item xs={3} sm={3} className={classes.profilePic}>
```

Comment:
btw, it doesn't make sense to specify `xs` and `sm` if they're going to be the same.

Reply:
Ok removed them both or am I suppose to only keep `sm={3}` ?